src/stylesheets/elements/typography/_links.scss

```diff
@@ -7,15 +7,9 @@
 // [href^='https:']:not([href*='my-domain.com'])
 
 .usa-link--external {
-  @include external-link(
-    usa-icons-bg/launch--blue-60v,
-    usa-icons-bg/launch--blue-70v
-  );
+  @include external-link();
 
   &.usa-link--alt {
-    @include external-link(
-      usa-icons-bg/launch--gray-5,
-      usa-icons-bg/launch--white
-    );
+    @include external-link("base-darker");
```

Comment:
> Should the expected value be something other than `currentColor` based on the SCSS below?

The value here is forwarded along as the `$contrast-bg` value for `add-color-icon`'s IE11 fallback. I selected it based on it being the background color for `.usa-dark-background`:

https://github.com/uswds/uswds/blob/1908d1391bc59410624ca1934cc70b7404e8f443/src/stylesheets/elements/typography/_content.scss#L38-L40

Maybe it would be clearer if I updated this to be a keyword parameter, so that it's not confused with the color of the icon itself?

```suggestion
    @include external-link($contrast-bg: "base-darker");
```

Reply:
>The value here is forwarded along as the $contrast-bg value for add-color-icon's IE11 fallback. I selected it based on it being the background color for .usa-dark-background:

I see. Initially thought it was defining the background to set the accessible link color. Because I tried setting `$contrast-bg` and the background to `primary-vivid` and was expecting the link color to change based on the new background color.

Reply:
> I see. Initially thought it was defining the background to set the accessible link color. Because I tried setting `$contrast-bg` and the background to `primary-vivid` and was expecting the link color to change based on the new background color.

As I see it, the expectation with these changes is that the icon follows whatever text color happens to be applied to the region, and it would be expected the text color contrast is assigned appropriately. The argument being passed here would only exist to maintain the existing behavior for the IE11 fallback of showing a white icon on dark background, and black icon on light background.

I'm not super familiar with the contrasting color behavior, so let me know if this isn't behaving the way you'd expect and if I should change it.

Reply:
In 872dc9e, I updated to use the named argument syntax just to make it extra clear what's being passed.